documentation/staging/content/samples/azure-kubernetes-service/domain-on-pv.md

```diff
@@ -574,22 +567,27 @@ In the example, the application address is: `http://52.224.248.40:8001/testwebap
 
 The test application will list the server host and server IP on the page.
 
-#### Access WebLogic Server logs
+#### Validate NFS volume
 
-The logs are stored in the Azure file share. Follow these steps to access the log:
+There are several approaches to validate the NFS volume:
 
-1. Go to the [Azure Portal](https://ms.portal.azure.com).
-2. Go to your resource group.
-3. Open the storage account.
-4. In the "File service" section of the left panel, select File shares.
-5. Select the file share name (e.g. weblogic in this example).
-6. Select logs.
-7. Select domain1.
-8. WebLogic Server logs are listed in the folder.
+- Use Azure Storage bowser, make sure you have permission to access the NFS server, see [Azure Storage firewalls and virtual networks document](https://docs.microsoft.com/en-us/azure/storage/common/storage-network-security?tabs=azure-portal)
+- Mount the same NFS share in an existing virtual machine from Azure, access files from the mounted path, see [Mount Azure NFS file share to Linux](https://docs.microsoft.com/en-us/azure/storage/files/storage-files-how-to-mount-nfs-shares).
```

Comment:
Please make the same edits here that I suggested previously for these statements.

Reply:
See https://github.com/galiacheng/weblogic-kubernetes-operator/commit/2b18f06d51752b28a22ec323464107e1d3305788